### PR TITLE
Date formatting

### PR DESCRIPTION
**Pull request by jankrzempek**
https://trello.com/c/54dZ8Fsr/281-update-date-format-to-mm-dd-yy

### CODE REVIEW

File: AirCasting/SessionViews/SessionHeaderView.swift [hunk 1]
```diff
@@ -174,14 +174,10 @@ private extension SessionHeaderView {
...
         let string = DateIntervalFormatter().string(from: start, to: end)
         // Purpose of this is to use 24h format all the time (no matter system settings)
-        if TimeConverter.is24Hour() {
-            // We are checking if user system settings is set to be 24h format
-                // if so, no additional change is needed
-            let replacedString = string.replacingOccurrences(of: "—", with: "-")
-            return Text(replacedString)
-        } else {
+        // and to always use format MM/dd/yyyy which is handled by a function
             if string.contains("–") {
```
[the remark this answers is not in the document]
Comment (pawelgil) [reply AirCasting/SessionViews/SessionHeaderView.swift:181]: I also have some second thoughts here, I think I missed something during the initial review, sorry for that.
The point - is all this work needed at all? If we just need to have a constant format for this why won't we use the [dateTemplate](https://developer.apple.com/documentation/foundation/dateintervalformatter/1407373-datetemplate) for hour formatting and force `en_US` [locale](https://developer.apple.com/documentation/foundation/dateintervalformatter/1409992-locale) to force `MM/dd/yyyy` date formatting?  Like this:
``` Swift
let formatter = DateIntervalFormatter()
formatter.dateTemplate = "HH:mm"
formatter.locale = Locale(identifier: "en_US")
```

Example output for interval within same day: `15:06 – 23:06`
Example output for interval with different days: `9/14/2021, 15:06 – 9/22/2021, 15:06`

**Review by pawelgil — CHANGES_REQUESTED** [review]

File: AirCasting/SessionViews/SessionHeaderView.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (jankrzempek) [reply AirCasting/SessionViews/SessionHeaderView.swift:181]: I need to say that I've used this solution (maybe wrong) and it gave me 'Sep 27 2021' which is not exactly correct. I made reformat and could you please suggest what to do with this?

**Comment by pawelgil** [comment]
@jankrzempek after testing this for a while - it crashes in `swapDaysAndMonths` for habitatmap user. I also checked out my recommended solution and it doesn't produce the format you mentioned. It's exactly as I put it in my comment: `9/14/2021, 15:06 – 9/22/2021, 15:06`

File: AirCasting/SessionViews/SessionHeaderView.swift (hunk 1 above)
[the remark this answers is not in the document]
Comment (jankrzempek) [reply AirCasting/SessionViews/SessionHeaderView.swift]: I am in shock @pawelgil but it is so awesome.!🥳

File: AirCasting/Utils/TimeConversion.swift [hunk 2]
```diff
@@ -23,4 +23,19 @@ final class TimeConverter {
         let time24 = df.string(from: date!)
         return time24
     }
+    
+    static func swapDaysAndMonths(date: String) -> String {
```
Comment (pawelgil) [inline AirCasting/Utils/TimeConversion.swift:27]: I think this not needed anymore?

**Review by pawelgil — CHANGES_REQUESTED** [review]

**Pull request merged**